packages/web-components/@groupby/elements-sayt/test/unit/sayt.test.ts

```diff
@@ -1085,4 +1095,64 @@ describe('Sayt Component', () => {
       expect(result).to.be.an.instanceof(TemplateResult);
     });
   });
+
+  describe('updateSearchboxInputTerm()', () => {
+    let requestUpdateSearchTerm;
+
+    beforeEach(() => {
+      requestUpdateSearchTerm = spy();
+      stub(sayt, 'querySelector')
+        .withArgs('[data-gbe-ref="autocomplete"]')
+        .returns({ requestUpdateSearchTerm });
+    });
+
+    it('should call the requestUpdateSearchTerm method on autocomplete when enter is pressed and the event target is within the searchbox component', () => {
+      stub(sayt, 'nodeInSearchbox').returns(true);
+      sayt.visible = true;
+
+      sayt.updateSearchboxInputTerm({ key: 'Enter' });
+
+      expect(requestUpdateSearchTerm).to.be.called;
+    });
+
+    it('should not call the requestUpdateSearchTerm method on autocomplete if a key other than enter is pressed', () => {
+      stub(sayt, 'nodeInSearchbox').returns(true);
+      sayt.visible = true;
+
+      sayt.updateSearchboxInputTerm({ key: 'Z' });
+
+      expect(requestUpdateSearchTerm).to.not.be.called;
+    });
+
+    it('should not call the requestUpdateSearchTerm method on autocomplete if the event node is not a child of searchbox', () => {
```

Comment:
We want it to work when the target _is_ the searchbox, so saying it's a child doesn't work. Also, please try to be consistent in your wording between these related tests.

```suggestion
    it('should not call the requestUpdateSearchTerm method on autocomplete if the event target is not contained in searchbox', () => {
```